fuel-core/src/args.rs

```diff
@@ -64,8 +74,35 @@ pub struct Opt {
     pub predicates: bool,
 }
 
+#[derive(clap::Subcommand, Clone, Debug)]
+pub enum Snapshot {
+    Snapshot(SnapshotCommand),
+}
```

Comment:
[nit] This probably deserved it's own mod, as in `args/snapshot.rs`, can be put under effort of refactoring cli subcommand: https://github.com/FuelLabs/fuel-core/issues/457